cli/cli.ts

```diff
@@ -3584,6 +3584,7 @@ function internalStaticPkgAsync(builtPackaged: string, label: string, minify: bo
         pkgversion: "0.0.0",
         fileList: pxtFileList("node_modules/pxt-core/").concat(targetFileList()),
         localDir,
+        target: pxt.appTarget.id,
```

Comment:
Target is used only when localDir is valid. For example it is not valid for ghPages. 
I am  fixing the second comment